### DISCUSSION TITLE
How to disable SSL for OAuth requests?

### DISCUSSION THREAD

**Discussion by advortsov** [apache/superset#26662] [Q&A / Help]
I have a self-written outh provider that should connect to the superset through the OAUTH_PROVIDERS block.

At the stage of requesting to obtain a token, I encounter:

`superset_app | 2024-01-18 12:23:30,617:ERROR:flask_appbuilder.security.views:Error authorizing OAuth access token: HTTPSConnectionPool(host='192.168.0.107', port=8150): Max retries exceeded with url: /api/v1 /party/oauth-clients/authenticate (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1129)')))`

How do I allow self-signed certificates and how do I configure a superset for such a case? 
Or how can I get rid of ssl?

**Comment by Molochnikov** [comment]
Can anyone help to run superset oauth with self-signed certificate?

**Comment by maksvg8** [comment]
I have a similar error. Have you solved the problem?

**Comment by betodealmeida** [comment]
@dpgaspar might have some insight here.

**Comment by Gianluke** [comment]
I have the same error: any updates on this?
